src/templates/nssfcfg.conf.j2

```diff
@@ -0,0 +1,98 @@
+configuration:
+  nrfUri: {{ nrf_url }}
+  nsiList:
+  - nsiInformationList:
+	- nrfId: {{ nrf_url }}/nnrf-nfm/v1/nf-instances
+	  nsiId: 22
+	snssai:
+	  sd: "010203"
+	  sst: 1
+  nssfName: NSSF
+  sbi:
+	bindingIPv4: 0.0.0.0
+	port: {{ sbi_port }}
+	registerIPv4: {{ nssf_url }}
+	scheme: http
+  serviceNameList:
+  - nnssf-nsselection
+  - nnssf-nssaiavailability
+  supportedNssaiInPlmnList:
+  - plmnId:
+	  mcc: "208"
+	  mnc: "93"
+	supportedSnssaiList:
+	- sd: "010203"
+	  sst: 1
+  supportedPlmnList:
+  - mcc: "208"
+	mnc: "93"
```

Comment:
Do we want to keep these configs?

Reply:
probably they should come from the webui/simapp.
https://github.com/gruyaume/omec-simapp-operator/blob/main/src/files/default_config.yaml#L81

Reply:
I removed those configs, thanks!